packages/documentation/src/stories/getting-started/packages/components-angular/components-angular.stories.ts

```diff
@@ -3,7 +3,7 @@ import { MetaExtended } from '../../../../../types';
 
 const meta: MetaExtended = {
   id: '833ef689-a573-40f5-a6a6-30a999b94733',
-  title: 'Getting Started/Packages/Components for Angular',
+  title: 'Getting Started/Packages/Components-Angular',
```

Comment:
If you want to go this way, you should change the tile on the home page and the introduction page.

Reply:
Actually I prefer "Components for Angular", I just thought we decided to go with "Components-Angular".

Reply:
I thought for a while that you've changed your mind about it. Then let's go with "for"